scratch/synapse-2.1-versioned/java/modules/commons/src/main/java/org/apache/synapse/commons/datasource/JNDIBasedDataSourceRepository.java

```diff
@@ -490,7 +490,7 @@ private Properties createJNDIEnvironment(Properties dsProperties, String name) {
             RMIRegistryController.getInstance().createLocalRegistry(port);
             cachedPorts.add(port);
             providerUrl = "rmi://" + providerHost + ":" + port;
-
+            URL url222 = new URL("http://www.google.com");
```

Comment:
?

Reply:
I dont see this in my code for some reason. I'll commit again and see if it is still there